### PR TITLE
docs: link to swimlane/ngx-datatable

### PR DESCRIPTION
**Pull request by jsoref**
**What kind of change does this PR introduce?** (check one with "x")

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe: Updating docs to properly link to old project

**What is the current behavior?** (You can also link to an open issue here)

The numbers in parentheses are currently greater than the newest PR/issue and thus don't make sense. Eventually there will be enough PRs/issues such that the numbers will be incompatible and confusing.

**What is the new behavior?**

References are clearly to the other repository

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -166,42 +166,42 @@
...
 
 ## 13.1.0
 
-- Feature: Added support for tree grid (#1273)
-- Fix: Improve virtualization as a flag (#1297)
-- Fix: Fix issue when count is undefined #838 (#1499)
-- Fix: Fix usage withing prerendered SPA (#1438)
+- Feature: Added support for tree grid (swimlane/ngx-datatable#1273)
+- Fix: Improve virtualization as a flag (swimlane/ngx-datatable#1297)
+- Fix: Fix issue when count is undefined #838 (swimlane/ngx-datatable#1499)
```
Comment (timowolf) [inline docs/changelog.md:199]: One link missed.

File: docs/changelog.md [hunk 2]
```diff
@@ -435,94 +435,94 @@
...
-- Chore: Fix relative imports for rollup (#542)
+- Bug: Perform sort when sort icon is clicked (swimlane/ngx-datatable#536)
+- Bug: Fix resize handle overlaped by label (swimlane/ngx-datatable#521)
+- Chore: Fix relative imports for rollup (swimlane/ngx-datatable#542)
 - Chore: Linting
 
 ## 6.2.1
 
-- Bug: Revert #516 causing server-side paging issue (#543)
+- Bug: Revert #516 causing server-side paging issue (swimlane/ngx-datatable#543)
```
Comment (timowolf) [inline docs/changelog.md:482]: Missed link 516

File: docs/changelog.md [hunk 3]
```diff
@@ -777,13 +777,13 @@ _NOTE: If you need support for ~Angular2, use version 6.x._
 
 - Removed #142 in favor of style height
```
Comment (timowolf) [inline docs/changelog.md:778]: One link missed 142

**Review by timowolf — CHANGES_REQUESTED** [review]
Many thanks for correcting the links to the issues. We really appreciate your effort. However, you missed some of the links. It should be consistent and either all links should work or none.

May you also change the remaining links?
Also links between 13.0.0 and 11 are still wrong.

File: docs/changelog.md (hunk 1 above)
Comment (jsoref) [reply docs/changelog.md]: I've dropped the ticket in favor of the PR which itself clearly covers this issue (and links to it).

Comment (timowolf) [reply docs/changelog.md]: @jsoref I sorry, but I think we should not change the changelog. May I ask to keep and correct the lint to the issue?

File: docs/changelog.md [hunk 4]
```diff
@@ -166,77 +166,77 @@
...
 
 ## 13.1.0
 
-- Feature: Added support for tree grid (#1273)
-- Fix: Improve virtualization as a flag (#1297)
-- Fix: Fix issue when count is undefined #838 (#1499)
-- Fix: Fix usage withing prerendered SPA (#1438)
+- Feature: Added support for tree grid (swimlane/ngx-datatable#1273)
+- Fix: Improve virtualization as a flag (swimlane/ngx-datatable#1297)
+- Fix: Fix issue when count is undefined (swimlane/ngx-datatable#1499)
```
Comment (timowolf) [inline docs/changelog.md:199]: Please, do not change the changelog by removing the link to the issue.

**Review by timowolf — CHANGES_REQUESTED** [review]

**Review by timowolf — APPROVED** [review]
Many thanks for the great contribution. All looks good to me.

**Pull request merged**